### ISSUE TITLE
Flask-WTF >= 0.14.2: Form validation fails, User is not registered

### ISSUE THREAD

**Issue by tobiasfrust** [tobiasfrust/shibboleth-authenticator#6]
When the `authorized_signup_handler`-function is called, the registration of the user fails due to a ValidationError of the created form. The relevant lines are [here](https://github.com/tobiasfrust/shibboleth-authenticator/blob/c3c0d19f3eaf622973cdaae41df45846ab3a4747/shibboleth_authenticator/handlers.py#L71-L77).
`form.to_dict()` results in the following output:
```python
{
    "email": "test@test.de",
    "profile": {
        "csrf_token": "",
        "username": "test1234",
        "full_name": "Full Name"
    }
}
```
`csrf_token` should not be present. It should already be deactived by `invenio_oauthclient.utils:disable_csrf`.
It is to be investigated what happens.

It seems to be introduced in version v0.14.2 of Flask-WTF in this [commit](https://github.com/lepture/flask-wtf/commit/42befd0420d1f4896a70339de3d474044461a1c9#diff-069d3a12d01a20991093d271171e770dL165).

**Comment by tobiasfrust** [comment]
Probably the same issue appears in this [issue](https://github.com/inveniosoftware/invenio-communities/issues/102).
It seems to be introduced in version v0.14.2 of Flask-WTF

**Comment by tobiasfrust** [comment]
fixed by daec3ab19a1647d1de98ce1fc722447707aa6b7f
